.prettierrc.yml

```diff
@@ -0,0 +1,5 @@
+printWidth: 80
```

Comment:
In your commit message you suggest 100:

> I propose to use the Airbnb standard which recommends to use 100 characters per line.

Reply:
Honestly, I prefer 100 characters per line, 80 characters is a little bit strict sometimes. But I saw that in the richie codebase, the rule of 80 characters is used so I change it afterwards.